flow/Stats.h

```diff
@@ -39,13 +39,40 @@ MyCounters() : foo("foo", cc), bar("bar", cc), baz("baz", cc) {}
 #include "flow/TDMetric.actor.h"
 
 struct TimedRequest {
```

Comment:
```suggestion
class TimedRequest {
```
If it has private fields I think it should be a class...